trino-connector/src/main/java/com/datastrato/gravitino/trino/connector/GravitinoMetadata.java

```diff
@@ -174,9 +175,9 @@ public ColumnMetadata getColumnMetadata(
 
   @Override
   public void createTable(
-      ConnectorSession session, ConnectorTableMetadata tableMetadata, boolean ignoreExisting) {
+      ConnectorSession session, ConnectorTableMetadata tableMetadata, SaveMode saveMode) {
     GravitinoTable table = metadataAdapter.createTable(tableMetadata);
-    catalogConnectorMetadata.createTable(table);
+    catalogConnectorMetadata.createTable(table, saveMode == SaveMode.IGNORE);
```

Comment:
What's the meaning of `SaveMode.IGNORE`?

Reply:
`saveMode == SaveMode.IGNORE ` it means `create table if not exist`